qt/scientific_interfaces/ISISReflectometry/test/Preview/PreviewModelTest.h

```diff
@@ -83,14 +83,6 @@ class PreviewModelTest : public CxxTest::TestSuite {
     TS_ASSERT_EQUALS(inputRoi, *model.getSelectedBanks())
   }
 
-  void test_set_selected_signal_region_converts_to_processing_instructions_string() {
```

Comment:
Sorry if the answer to this question is obvious, but was there a reason for removing this test?